CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
nit on the commit title: `fix(mmds)` or `fix(http)`

Reply:
This change is made on micro-http. It should not mention mmds and it's obvious to change related to http.

Reply:
yep, sorry I thought this was the firecracker repo.